lib/pbench/server/auth/__init__.py

```diff
@@ -456,8 +458,8 @@ def _delete(self, path: str, data: Dict = {}, **kwargs) -> requests.Response:
         Resource at a given path.
         Args:
             path: Path for the request.
-            data: data to post with the request
-            kwargs: Params dict to send with GET request
+            data: data to send with the request
+            kwargs: Params dict to send with DELETE request
```

Comment:
Ha; I thought it looked odd that you were matching the verb in the "data" description with the HTTP method, but although it kinda works for `POST` and `PUT` it obviously doesn't here. You could actually use "data to send" for all of them, which would be a little better... but "JSON request body" would be a more useful description.